Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:2
+
+RUN apt-get update && apt-get install -y postgresql-client
+
+COPY requirements.txt /usr/src/app/
+WORKDIR /usr/src/app
+RUN pip install -r requirements.txt
+
+EXPOSE 8000
+
+COPY ampcrowd/docker-entrypoint.sh /usr/src/app/ampcrowd
+ENTRYPOINT ["bash", "ampcrowd/docker-entrypoint.sh"]
+CMD ["-d"]
```

Comment:
`foreground` sounds good.

Celery is the python amqp client that sits between Django and rabbitmq. If we're not in debug mode, I run it in daemon mode with `celeryd_detach`, but when debugging, it can be useful to have it dump output to stdout, so I just run the foreground process `celeryd`. Since this isn't needed to keep Docker up, and is really only useful when debugging issues with celery, I think the existing behavior is fine: let the `debug` flag pick the execution strategy, and ignore `foreground`.